runtime/src/main/resources/application-terraformboot-local.properties

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
we can just have one profile - application-terraformboot.properties and enable it by default. 
Pls also check once with WangLi.. i think he also has added a new profile in this PR.